lca-cli/postpivot/postpivot.go

```diff
@@ -51,8 +53,11 @@ func NewPostPivot(scheme *runtime.Scheme, log *logrus.Logger, ops ops.Ops, authF
 }
 
 const (
-	nodeIpFile       = "/run/nodeip-configuration/primary-ip"
-	dnsmasqOverrides = "/etc/default/sno_dnsmasq_configuration_overrides"
+	nodeIpFile            = "/run/nodeip-configuration/primary-ip"
+	dnsmasqOverrides      = "/etc/default/sno_dnsmasq_configuration_overrides"
+	sshKeyEarlyAccessFile = "/home/core/.ssh/authorized_keys.d/ib-early-access"
```

Comment:
Why not use the same file?
/home/core/.ssh/authorized_keys.d/ignition

Reply:
Cause /home/core/.ssh/authorized_keys.d/ignition is created by mco and i don't want to touch files created by it.
This file will be deleted by mco at the end in anycase